src/pages/EntityDetails/Model/ActionLogs/ActionLogs.tsx

```diff
@@ -102,6 +124,39 @@ export default function ActionLogs() {
 
   const applicationList = () => Object.keys(modelStatusData.applications);
 
+  const OutputSelector = ({
+    onSelect,
+    type,
+    hasStdOut,
+    hasStdErr,
+  }: {
+    onSelect: (type: string) => void;
+    type: string;
+    hasStdOut: boolean;
+    hasStdErr: boolean;
+  }) => (
+    <ContextualMenu
+      hasToggleIcon
+      toggleLabel={type === "STDOUT/STDERR" ? "Output" : type}
+      links={[
+        {
+          children: "STDOUT/STDERR",
+          onClick: () => onSelect("STDOUT/STDERR"),
+        },
+        {
+          children: "STDERR",
+          onClick: () => onSelect("STDERR"),
```

Comment:
These states rely on specific strings so it'd be a little less fragile if these were to use an enum, something like:

```typescript
enum Output {
    ALL = "STDOUT/STDERR",
    STDERR = "STDERR",
    STDOUT = "STDOUT",
}
```

Reply:
Good idea :)